bin/check_fixmystreet

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env perl
+
+use strict;
+use lib '/usr/lib/nagios/plugins';
+use utils qw(%ERRORS);
+use Getopt::Long;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../../fixmystreet/setenv.pl";
+}
+
+my $reports = 50;
+my $updates = 0;
+
+GetOptions(
+    "reports=i" => \$reports,
+    "updates=i" => \$updates,
+);
+
+sub unsent_report_count {
+
+    use FixMyStreet::DB;
+    use FixMyStreet::Script::Reports;
+
+    my $params = FixMyStreet::Script::Reports::construct_query(1);
+    my $unsent = FixMyStreet::DB->resultset('Problem')
+       ->search({ confirmed => { '<', \"current_timestamp - '5 minutes'::interval" } })
+       ->search($params);
+
+    return $unsent->count;
+
+}
+
+sub pending_update_count {
+
+    use FixMyStreet::DB;
+    use Open311::PostServiceRequestUpdates;
+
+    my $updates = Open311::PostServiceRequestUpdates->new( verbose => 1 );
+    my $bodies = $updates->fetch_bodies;
+    my $params = $updates->construct_query(1);
+
+    my $u = FixMyStreet::DB->resultset("Comment")
+       ->to_body([ keys %$bodies ])
+       ->search({ "me.confirmed" => { '<', \"current_timestamp - '30 minutes'::interval" } }) # ignore any made in the last half hour
```

Comment:
I know I suggested this line, but now I am wondering why. I guess it's if we switched back to cron from the daemon? But really, you could change this to 5 minutes to match reporting perhaps.

Reply:
Seems sensible, yes.